.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Introduction 
+
+This outlines how to propose a change to admiral . For more detailed info about contributing to admiral, and other pharmaverse packages, please see the [development process guide](https://pharmaverse.github.io/admiral/articles/development_process.html) as well as other Developer Guides in the Articles section of [admiral website](https://pharmaverse.github.io/admiral/index.html)
```

Comment:
what's the convention for "admiral" - is it a text as it is or rather `{admiral}` as in the next paragraph?

Reply:
i think we are using {admiral} by convention - although I think the tidyverse style guide advises against this.